### PR TITLE
converted anon functions to lamba

### PR DESCRIPTION
**Pull request by Marc-Week**
Converted the function () to => lambda

### CODE REVIEW

**Comment by kathtran** [comment]
LGTM

File: test/lib/pagination.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ let startAt = 0;
 let page = 0;
 let maxPossible = Number.MAX_SAFE_INTEGER;
 
-let rejectedRequestStub = function (options, callback) {
+let rejectedRequestStub = (options, callback) => {
   process.nextTick(function () {
```
Comment (kathtran) [inline test/lib/pagination.js]: This

File: test/routes/index.js [hunk 2]
```diff
@@ -84,12 +84,12 @@ describe('index', function () {
     });
   });
 
-  describe.skip('post request', function () {
+  describe.skip('post request', () => {
     'use strict';
 
-    credentialFixtureCases.forEach(function (fixture) {
+    credentialFixtureCases.forEach((fixture) => {
```
Comment (RickyV33) [inline test/routes/index.js]: this

File: test/routes/projects.js [hunk 3]
```diff
@@ -12,15 +12,15 @@ let join = require('path').join;
 
 chai.use(chaiHttp);
 
-describe('projects', function () {
-  describe.skip('get request', function () {
+describe('projects', () => {
+  describe.skip('get request', () => {
     // Test #1: verify projects list view is rendered successfully as is documented in views/projects.ejs
-    it('should render successfully with status 200', function (done) {
+    it('should render successfully with status 200', (done) => {
```
Comment (RickyV33) [inline test/routes/projects.js]: this

**Pull request merged**